kinto_wizard/kinto2yaml.py

```diff
@@ -27,19 +34,26 @@ def introspect_bucket(client, bid, full=False):
     if len(permissions) == 0:
         logger.warn('Could not read permissions of bucket {!r}'.format(bid))
 
-    collections = client.get_collections(bucket=bid)
-    groups = client.get_groups(bucket=bid)
-    result = {
-        'permissions': _sorted_principals(permissions),
-        'collections': {
-            collection['id']: introspect_collection(client, bid, collection['id'], full=full)
-            for collection in collections
-        },
-        'groups': {
-            group['id']: introspect_group(client, bid, group['id'], full=full)
-            for group in groups
+    if collection:
+        result = {
+            'permissions': _sorted_principals(permissions),
+            'collections': {collection: introspect_collection(client, bid, collection, full=full)},
+            'groups': {}
```

Comment:
👍 Yes I was wondering the same. I kept it like that because I was able to use the same dump file in the test. But I guess being lazy is not an argument 🗡